### ISSUE TITLE
keras分支的损失函数无限减小 最后程序报错

### ISSUE THREAD

**Issue by primejava** [yinguobing/cnn-facial-landmark#80]
`ETA: 3:06 - loss: 1.8145e-04 - mean_squared_error: 1.8145e-04WARNING:tensorflow:Method on_batch_end() is slow compared to the batch update (4.427386). Check your callbacks.`

**Comment by primejava** [comment]
请问可以给keras版本写一篇使用文章吗 我觉得可能是某个步骤我做错了 因为我在使用自己的数据集重新进行训练

**Comment by yinguobing** [comment]
Issue是不是已经解决了？从报错信息来看是callback的运行时间过长，需要具体分析。
